rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/telemetry/TbMsgAttributesNode.java

```diff
@@ -64,15 +67,15 @@
 )
 public class TbMsgAttributesNode implements TbNode {
 
+    static final String NOTIFY_DEVICE_KEY = "notifyDevice";
+    static final String SEND_ATTRIBUTES_UPDATED_NOTIFICATION_KEY = "sendAttributesUpdatedNotification";
     static final String UPDATE_ATTRIBUTES_ONLY_ON_VALUE_CHANGE_KEY = "updateAttributesOnlyOnValueChange";
+
     private TbMsgAttributesNodeConfiguration config;
 
     @Override
     public void init(TbContext ctx, TbNodeConfiguration configuration) throws TbNodeException {
         this.config = TbNodeUtils.convert(configuration, TbMsgAttributesNodeConfiguration.class);
-        if (config.getNotifyDevice() == null) {
```

Comment:
Why it is gone?

Reply:
See my [comment](https://github.com/thingsboard/thingsboard/pull/9652#discussion_r1397329492)